src/routes/marketlist/store/models/bounds.js

```diff
@@ -0,0 +1,9 @@
+import { Record } from 'immutable'
```

Comment:
@mmvsha  @andre-meyer 
Maybe should I introduce this record inside ```store/models/market.js```?

Reply:
I'd say yes, the file is not big and it's not a problem to have multiple entities in the same file
@apanizo

Reply:
👍